--- setup_kinetic.bash	2018-12-03 22:12:08.064222522 +0800
+++ setup_kinetic_new.bash	2018-12-03 22:14:22.538941279 +0800
@@ -78,8 +78,10 @@
 cd ..
 catkin_make --pkg mav_msgs
 source devel/setup.bash
+catkin_make --pkg driver_base
+source devel/setup.bash
 catkin_make -j 1
-bash -c 'echo source `pwd`/devel/setup.bash >> ~/.bashrc'
+#bash -c 'echo source `pwd`/devel/setup.bash >> ~/.bashrc'
 echo "## ROS workspace compiled ##"
 
 # add own models path to gazebo models path
